src/reducers/Entities/NameNoteReducer.js

```diff
@@ -51,7 +51,7 @@ export const NameNoteReducer = (state = initialState(), action) => {
       const { nameNote } = payload;
       const { id } = nameNote;
 
-      const newCreatingById = { ...creatingById, [id]: nameNote.toJSON(), creatingId: '' };
+      const newCreatingById = { ...creatingById, [id]: nameNote, creatingId: '' };
 
       return { ...state, creatingById: newCreatingById };
```

Comment:
good point, I haven't used the `id` fields. will review

Reply:
fixed this one in 2a3814f9
Created #3671 to tidy up the nameNote state - there were a few changes and I wanted them recorded rather than appended to this PR